partner_communication_switzerland/models/partner_communication.py

```diff
@@ -111,6 +111,34 @@ def get_birthday_bvr(self):
                 birthday_gift, background)
         return attachments
 
+    def get_gift_bvr(self):
+        """
+        Attach gift slip with background for sending by e-mail
+        :return: dict {attachment_name: [report_name, pdf_data]}
+        """
+        self.ensure_one()
+        attachments = dict()
+        background = self.send_mode and 'physical' not in self.send_mode
+        sponsorships = self.get_objects()
+        bvr_other = False
+        bvr_gifts = False
+        if sponsorships:
+            if sponsorships.payment_mode_id.id == 10:
+                if sponsorships.partner_id == sponsorships.correspondent_id:
```

Comment:
You cannot directly access a field of sponsorships. It will fail when more than one sponsorship is attached to the communicaton. Use function mapped to retrieve the list of payment_mode_ids and check if 10 is in the list